src/libs/API/parameters/UpdateConnectionConfigParams.ts

```diff
@@ -0,0 +1,10 @@
+import type {ConnectionConfig, ConnectionName} from '@src/types/onyx/Policy';
+
+type UpdateConnectionConfigParams = {
+    policyId: string;
+    connectionName: ConnectionName;
+    config: Partial<ConnectionConfig>;
+    idempotencyKey: string;
+};
```

Comment:
I think these need to be updated according to: https://github.com/Expensify/App/pull/38006/files#r1542171042

Reply:
updated https://github.com/Expensify/App/pull/38006/commits/14ae20a2d427e1f360d7638ef804685aeb797624